### ISSUE TITLE
"le register --force" results in duplicate host name

### ISSUE THREAD

**Issue by markstos** [logentries/le#103]
On initial setup, the hostname didn't come through correctly. 

I first tried `le register --name=foo --hostname=food.example.com to fix it.` as I didn't see a way to do this through the website. 

That failed, so I added the "force" option, which is documented to " override current registration". That sounded perfect.  However, Adding --force did not replace the current registration, it added an additional host entry on the website, so I still have a stale host entry I'm not sure how to remove.

**Comment by vilda** [comment]
Yep Mark, that's correct behavior. The register command registers a new host. The agent checks if it's not registered already, that's why --force is required. It's not trying to update any previous registration. Do you think "override current registration" is not clear and should be changed?

**Comment by markstos** [comment]
Yes, I think updating that wording would be helpful. Thanks.

**Comment by porn** [comment]
Then is there any way how to "register" new host under existing logentries set? In my case I'd like to register new hosts under the existing log set (let's say _production_) each time I create a new machines and drop the old ones.

**Comment by StephenHynes7** [comment]
@porn best way to do this would be to use https://github.com/logentries/le#follow-log-files-through-server-side-configuration and https://github.com/logentries/le#follow-log-files-through-your-configuration-file. You would then set `destination` to the existing logsets in your account.

What you would need to make sure though is that the logs are Token TCP logs and not HTTP PUT logs.

The alternative to this approach would just to add `--host-key` to your register call and use the the existing host key which can be retrieved from the UI.

**Comment by porn** [comment]
Thanks @StephenHynes7!
